### PR TITLE
refactor: Improve db inspection

### PR DESCRIPTION
**Pull request by isaquealves**
- Add support to postgresql numeric type.
- Improve field configuration handling for numeric and decimal types

### CODE REVIEW

File: aerich/inspect/__init__.py [hunk 1]
```diff
@@ -30,8 +30,13 @@ def translate(self) -> dict:
...
             length = f"max_length={self.length}, "
-        if self.data_type == "decimal":
-            length = f"max_digits={self.max_digits}, decimal_places={self.decimal_places}, "
+        if self.data_type in ["decimal", "numeric"]:
+            length_parts = []
+            if self.max_digits:
+                length_parts.append(f"max_digits={self.max_digits}")
+            if self.decimal_places:
+                length_parts.append(f"decimal_places={self.decimal_places}")
+            length = "".join(length_parts)
```
Comment (long2ice) [inline aerich/inspect/__init__.py]: This should be `", ".join`

**Review by long2ice — APPROVED** [review]

**Comment by long2ice** [comment]
Thanks! Please update the changelog

File: aerich/inspect/__init__.py [hunk 2]
```diff
@@ -30,8 +30,13 @@ def translate(self) -> dict:
...
             length = f"max_length={self.length}, "
-        if self.data_type == "decimal":
-            length = f"max_digits={self.max_digits}, decimal_places={self.decimal_places}, "
+        if self.data_type in ["decimal", "numeric"]:
+            length_parts = []
+            if self.max_digits:
+                length_parts.append(f"max_digits={self.max_digits}")
+            if self.decimal_places:
+                length_parts.append(f"decimal_places={self.decimal_places}")
+            length = ",".join(length_parts)
```
Comment (long2ice) [inline aerich/inspect/__init__.py]: A space is need

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
 # ChangeLog
 
 ## 0.6
-
+### 0.6.4
+- Improve `inspectdb` adding support to `postgresql::numeric` data type
```
Comment (long2ice) [inline CHANGELOG.md:7]: Need a newline here

**Review by long2ice — APPROVED** [review]

File: aerich/inspect/__init__.py (hunk 2 above)
Comment (isaquealves) [reply aerich/inspect/__init__.py]: done

File: CHANGELOG.md (hunk 3 above)
Comment (isaquealves) [reply CHANGELOG.md:7]: done

File: aerich/inspect/__init__.py (hunk 1 above)
Comment (isaquealves) [reply aerich/inspect/__init__.py]: done

**Review by long2ice — APPROVED** [review]

**Comment by isaquealves** [comment]
@long2ice just fixed some tests

**Review by long2ice — APPROVED** [review]

**Comment by long2ice** [comment]
Looks like style need fix

**Comment by isaquealves** [comment]
> Looks like style need fix
fixed, @long2ice

**Pull request merged**